core/src/main/java/org/mule/runtime/core/internal/policy/AbstractCompositePolicy.java

```diff
@@ -27,11 +27,10 @@
  * Abstract implementation that performs the chaining of a set of policies and the {@link Processor} being intercepted.
  *
  * @param <ParametersTransformer> the type of the function parameters transformer.
- * @param <ParametersProcessor> the type of the parameters processor that provides access to the initial value of the parameters.
  *
  * @since 4.0
  */
-public abstract class AbstractCompositePolicy<ParametersTransformer, Subject> {
+public abstract class AbstractCompositePolicy<ParametersTransformer> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. ([squid:S00119](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS00119))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7760&issues=AWnLDJg_otIXnmsUm4Yh&open=AWnLDJg_otIXnmsUm4Yh)

[ISSUE_KEY]: AWnLDJg_otIXnmsUm4Yh